src/pipeline.cc

```diff
@@ -38,6 +38,9 @@
 #elif defined(__APPLE__)
 #define STAT64_STRUCT stat
 #define STAT64_FUNCTION stat
+#elif defined(__FreeBSD__)
```

Comment:
Is it worth expanding this to include variants, e.g.:
```c
#elif defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__DragonFly__)
```

Reply:
Ok I checked `sys/stat.h` on OpenBSD, NetBSD and DragonFly, they're all stat instead of stat64. Wanted to include all of them but I wasn't sure if they're all stat.

Reply:
I made the change.